On some Mediatek kernel this seems to help with incorrect path "./tools/tools"
---

diff --git a/tools/scripts/Makefile.include b/tools/scripts/Makefile.include
index 8abbef164b4e..7a2f1510a630 100644
--- a/tools/scripts/Makefile.include
+++ b/tools/scripts/Makefile.include
@@ -2,7 +2,7 @@ ifneq ($(O),)
 ifeq ($(origin O), command line)
 	dummy := $(if $(shell test -d $(O) || echo $(O)),$(error O=$(O) does not exist),)
 	ABSOLUTE_O := $(shell cd $(O) ; pwd)
-	OUTPUT := $(ABSOLUTE_O)/$(if $(subdir),$(subdir)/)
+	OUTPUT := $(ABSOLUTE_O)/
 	COMMAND_O := O=$(ABSOLUTE_O)
 ifeq ($(objtree),)
 	objtree := $(O)
